frontend/lib/src/components/shared/StreamlitMarkdown/styled-components.ts

```diff
@@ -218,3 +218,7 @@ export const StyledDivider = styled.hr<StyledDividerProps>(
     }
   }
 )
+
+export const StyledPreWrapper = styled.div(({ theme }) => ({
+  marginBottom: theme.spacing.lg,
```

Comment:
nit: can you add a comment here that this is set to provide spacing between pre-elements inside markdown similar to our gap spacing for elements?

Reply:
great point! Will add it